### PR TITLE
refine InstructorCourseAddAction Slice

### PR DESCRIPTION
**Pull request by bamboo3250**
redo InstructorCourseAddAction Slice

### CODE REVIEW

**Comment by bamboo3250** [comment]
redo #3416

**Comment by damithc** [comment]
I checked the code for style. Seems fine. 
Shouldn't this slice involve CoursesLogic, CoursesDb etc? Or there were nothing to do in those files? 
What about jsp files?
Did you check test coverage?

**Comment by bamboo3250** [comment]
@damithc about CoursesLogic, CoursesDb and other levels related to course, I'm reviewing them in the next slice. 
What should I do with the branches that aren't reached when doing test coverage?

**Comment by damithc** [comment]
Noted. For branches not reached, try to cover them be adding more tests or modifying existing tests.

**Comment by bamboo3250** [comment]
@damithc I added a test case to reach all the branches. Please check the code style, especially the instructorCourseEdit.jsp.

dev green

**Comment by damithc** [comment]
Some of the line breaks may not be necessary if we go by the new 120 char limit for a line. Other than that, I did not spot any problems. After removing unnecessary line breaks, pass to Hong Jin for a final review.

**Comment by bamboo3250** [comment]
Just to confirm, the 120 char limit is only applied for .java files, isn't it? I think so because it may not look nice for .jsp files due to their huge hierarchy structure.

**Comment by damithc** [comment]
Yes, JSP files need not follow that limit.

**Comment by bamboo3250** [comment]
@kanghj please review this one

File: src/test/java/teammates/test/cases/ui/InstructorCourseAddActionTest.java [hunk 1]
```diff
@@ -115,34 +108,63 @@ public void testExecute() throws Exception{
...
+        expectedLogMessage = "TEAMMATESLOG|||instructorCourseAdd|||instructorCourseAdd|||true|||Instructor(M)|||"
+                             + "Instructor 1 of Course 1|||idOfInstructor1OfCourse1|||instr1@course1.tmt|||"
+                             + "Course added : ticac.tpa2.id<br>Total courses: 1|||/page/instructorCourseAdd";
         assertEquals(expectedLogMessage, addAction.getLogMessage());
         
         // delete the new course
         CoursesLogic.inst().deleteCourseCascade("ticac.tpa2.id");
+        
+        ______TS("Test archived Courses");
+        InstructorAttributes instructorOfArchivedCourse = dataBundle.instructors.get("instructorOfArchivedCourse");
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/InstructorCourseAddActionTest.java]: There is a comment at at the start of the file that says that the path for checking archived courses will not be covered, you can remove/modify that comment now that you have added this case

**Pull request merged**